documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -99,7 +107,7 @@ The options for the touchscreen are:
 
 === X11-only features
```

Comment:
Should explain why you might be running X11. Either older Pi hardware — although in time Bookworm should be moving to Wayland for all Pi hardware — or an older version of the OS.

Reply:
I'm not sure if labelling these as "X11-only features" is the angle we want to go for here (but this is obviously a decision for @aallan to make). IIRC these are features which we also **want** to see in Wayland (in future), but which simply haven't been implemented yet. :shrug: :confused:

Reply:
For stuff that only works in X11 or differently in X11 and not in Wayland, or only in a previous version of the OS, as I mentioned before, I want these hared off into a separate NOTE block. Preferably a short standalone, minimally documented one with *just the incantations* not masses of explanatory text around it.

Reply:
After throwing it in a note block I didn't like its current location. Moved it to the front of the Legacy section instead. Let me know if that works better.